dss/events/handlers/index.py

```diff
@@ -4,33 +4,46 @@
 from urllib.parse import unquote
 
 import boto3
+import botocore
 
 from ... import DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE
-from ...util import connect_elasticsearch
+from ...hcablobstore import BundleMetadata, BundleFileMetadata
+from ...util import connect_elasticsearch, create_blob_key
 
 DSS_BUNDLE_KEY_REGEX = r"^bundles/[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}\..+$"
 
-#
-# Lambda function for DSS indexing
-#
+"""
```

Comment:
I don't recall if python docblocks need to be after the class declaration or before.  Might want to check that.